tests/helpers.py

```diff
@@ -218,9 +225,28 @@ def create_multilpe_projects(number_of_project):
     return project_objs
 
 
+def default_secret():
```

Comment:
As mentioned in this [comment](https://github.com/red-hat-storage/ocs-ci/pull/965/files#r344012833), there is no single default secret for provisioning and mounting. Also there are seperate secrets for cephfs and rbd

Reply:
I will remove this secret function, I dont see the need for this.